### ISSUE TITLE
UX: Profile information shown in 2 different ways when the User is new and when he navigates to Profile settings

### ISSUE THREAD

**Issue by bevanamit** [theheapdump/kite-webapp#203]
Profile and broker information pages appear different for the first time and then clicking on Profile Settings page appears differently, Keeping it consistent would is good

![image](https://user-images.githubusercontent.com/5267251/102382426-773b2d80-3ff0-11eb-9019-d86e93c248a5.png)

@anmoldeep0123 provide ur inputs

**Comment by theheapdump** [comment]
i want first @kshitijbhardwaj  to comment

**Comment by kshitijbhardwaj** [comment]
Yes because we have designed it like that. Whats the issue in that?? This is as per design.

**Comment by kshitijbhardwaj** [comment]
Requesting to please close this.

**Comment by theheapdump** [comment]
Let's discuss this over a call @bevanamit
